css/partials/pages/_forms.scss

```diff
@@ -0,0 +1,92 @@
+.wpcf7 {
+
+  fieldset {
+    margin-top: 1.5rem;
+    margin-bottom: 1.5rem;
+    border: 1px solid #ccc;
+    padding: 1rem 1.5rem;
+    width:75%;
+  }
+
+  input[type="text"],
+  input[type="email"],
+  textarea {
+    background-color: #fff;
```

Comment:
This rule is ok if these styles wouldn't be inherited from the parent element, but if you can avoid styling elements directly (use classes when possible), all the better.